airsonic-main/src/main/java/org/airsonic/player/service/SecurityService.java

```diff
@@ -162,11 +162,20 @@ public User getUserByEmail(String email) {
         return userDao.getAllUsers();
     }
 
+    public String[] getAllUsernames() {
+        List<User> users = userDao.getAllUsers();
+        String[] usernames = new String[users.size()];
+        for (int i = 0; i < users.size(); i++) {
```

Comment:
nit: I would have used a `foreach` here.

Reply:
I would have too, except for-each doesn't provide the index, and since I'm building an array I need the index.